Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs

```diff
@@ -147,7 +147,7 @@ public void TestWithTor()
                 CoreNode node1 = builder.CreateStratisPowNode(true, fullNodeBuilder =>
                 {
                     fullNodeBuilder
-                        .UseConsensus()
+                        .UsePosConsensus()
```

Comment:
Should be UsePowConsensus here; TB runs on the BTC network at present

Reply:
done

Reply:
done